imports/plugins/core/system-info/server/no-meteor/schemas/schema.graphql

```diff
@@ -0,0 +1,27 @@
+extend type Query {
+  "SystemInformation object"
+  systemInformation(shopId: ID!): SystemInformation!
+}
+"Represents Reaction Plugin"
+type Plugin {
+  "name of plugin"
```

Comment:
All field descriptions should begin with a capital letter by convention. I guess our GraphQL linter does not check this.